### ISSUE TITLE
Need some way to tell Discord what command line parameters joining players need to use.

### ISSUE THREAD

**Issue by StrikerMan780** [discordapp/discord-rpc#46]
To quote my comment on issue #35: "Of course, I could avoid this, and any other issues altogether if there were a way to set what command line arguments are to be used when another player launches the game through Discord. (ie. Some way to tell their Discord client they'll need to launch the game with "/g mymod.pk3 /n:0 123.231.213.132", like some .partycmd field in the discordPresence struct or something.)"

In short, this feature, would render issue #35 a moot point for me.

**Comment by crmarsh** [comment]
No, I don't think we're going to support that, that's what the join/spectate secrets are for.

**Comment by StrikerMan780** [comment]
Damn, that makes things needlessly complicated.

**Comment by crmarsh** [comment]
I think both `needlessly` and `complicated` are arguable here; it sounds like what you really want for your use case is a frontend launcher program that handles matchmaking, then launches the game. If you set things up that way (which makes sense to me independent of Discord) it would then integrate with this very naturally.
